addons/website_slides/models/slide_slide_resource.py

```diff
@@ -18,6 +18,7 @@ class SlideResource(models.Model):
     data = fields.Binary('Resource', compute='_compute_reset_resources', store=True, readonly=False)
     file_name = fields.Char(store=True)
     link = fields.Char('Link', compute='_compute_reset_resources', store=True, readonly=False)
+    download_url = fields.Char(compute='_compute_download_url')
```

Comment:
can you add a label on the field ? (if we see it in technical view or something like that)